CLI Errors

I tried running the following commands before getting the following errors. What am I doing wrong here?

```
git clone git@github.com:robertDurst/stellar-vanity-address-generator.git
cd stellar-vanity-address-generator
cargo run -- -c=8 --postfix=p
```

```
charliefish@Charlies-MacBook-Pro stellar-vanity-address-generator % cargo run -- -c=8 --postfix=p
   Compiling stellar_vanity v0.5.0 (/Users/charliefish/stellar-vanity-address-generator)
error[E0277]: the trait bound `T: rand_core::CryptoRng` is not satisfied
   --> src/vanity_key/mod.rs:56:32
    |
51  |     T: Rng + CryptoRng,
    |                        - help: consider further restricting type parameter `T`: `, T: rand_core::CryptoRng`
...
56  |         Some(Keypair::generate(&mut self.rng))
    |                                ^^^^^^^^^^^^^ the trait `rand_core::CryptoRng` is not implemented for `T`
    | 
   ::: /Users/charliefish/.cargo/registry/src/github.com-1ecc6299db9ec823/ed25519-dalek-1.0.0-pre.3/src/ed25519.rs:130:12
    |
130 |         R: CryptoRng + RngCore,
    |            --------- required by this bound in `ed25519_dalek::Keypair::generate`

error[E0277]: the trait bound `T: rand_core::RngCore` is not satisfied
   --> src/vanity_key/mod.rs:56:32
    |
51  |     T: Rng + CryptoRng,
    |                        - help: consider further restricting type parameter `T`: `, T: rand_core::RngCore`
...
56  |         Some(Keypair::generate(&mut self.rng))
    |                                ^^^^^^^^^^^^^ the trait `rand_core::RngCore` is not implemented for `T`
    | 
   ::: /Users/charliefish/.cargo/registry/src/github.com-1ecc6299db9ec823/ed25519-dalek-1.0.0-pre.3/src/ed25519.rs:130:24
    |
130 |         R: CryptoRng + RngCore,
    |                        ------- required by this bound in `ed25519_dalek::Keypair::generate`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0277`.
error: could not compile `stellar_vanity`.

To learn more, run the command again with --verbose.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,7 +10,7 @@ repository = "https://github.com/robertDurst/stellar-vanity-address-generator"
 license-file = "LICENSE"
 
 [dependencies]
-rand = "0.6"
+rand = "0.7.3"
 ed25519-dalek = "1.0.0-pre.1"
 base32 = "0.3.1"
 crc16 = "0.3.4"

--- a/src/vanity_key/mod.rs
+++ b/src/vanity_key/mod.rs
@@ -40,7 +40,7 @@ where
 
 impl Default for AddressGenerator<OsRng> {
     fn default() -> Self {
-        let rng = OsRng::new().unwrap();
+        let rng = OsRng;
 
         AddressGenerator { rng }
     }

